### DISCUSSION TITLE
slf4j logger change

### DISCUSSION THREAD

**Discussion by Psijic** [mapsforge/vtm#1067] [Q&A] [answered]
Not sure if it's possible to change with external options:
`log = LoggerFactory.getLogger(OkHttpEngine.class);`
`slf4j` is not included in pure Java or Android project by default. And `Timber` is the standard de-facto for Android. So, how to change `slf4j`  to `Timber` in build options?

**Comment by devemux86** [comment, marked as answer]
VTM is a multi-platform library.
`slf4j` was chosen in the past by the former developers for its flexibility.

If we remove it we should return to the native `java.util.logging.Logger`.
(as it also does not need extra dependencies)

**Reply by Psijic** [reply]
Usually it's good to remove tech debt.
